src/main/java/com/softserve/academy/spaced/repetition/controller/CourseController.java

```diff
@@ -132,8 +132,7 @@ public ResponseEntity deleteGlobalCourse(@PathVariable Long courseId) throws Not
     }
 
     @DeleteMapping(value = "/api/cabinet/local/courses/{courseId}")
-    @PreAuthorize("hasPermission('COURSE','DELETE') &&" +
-            "@courseServiceImpl.getCourseById(#courseId).createdBy==principal.id")
```

Comment:
Now an user can delete the course he hasn't created.
I believe only admin can delete the course he hasn't created.
Could you please double check?